src/models/movie.js

```diff
@@ -0,0 +1,52 @@
+// Import the sequelize connection from config file 
+// Import datatypes from the sequelize library
+
+const sequalize = require('../configs/postgresqldb').sequalize;
+const DataTypes = require('sequelize');
+
+// Define the model by providing name of the table, it's columns, their datatypes and constraints.
+
+const Movie = sequalize.define('movie', {
```

Comment:
There seems to be creating relationships and how we design normalized tables. Can you think of Theatre, Movie, Show kind of structure and come up with db design for it?